static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss

```diff
@@ -1,31 +1,26 @@
 @mixin immersiveCommon() {
     .fc-item__kicker {
+        color: $news-garnett-highlight;
         display: block;
     }
 
+    .fc-item__headline {
+        color: #ffffff;
+    }
+
     .fc-item__container {
         overflow: hidden;
+        position: relative;
     }
 
-    .fc-item__content {
-        background-color: $immersive-garnett-main-1;
-    }
-
-    //Todo better way to get this immersive card type to set the height when sublinks. This is fixed height enough to fit two lines of sublink
-    &.fc-item--has-sublinks-3 .fc-item__content,
-    &.fc-item--has-sublinks-2 .fc-item__content {
-        @include mq($from: tablet) {
-            padding-bottom: $gs-gutter * 3.6;
-        }
+    // TODO: remove this temporary css fix
```

Comment:
Is this easy to do? What's blocking doing this now?

Reply:
Will take a look now 🥇

Reply:
Just to let you know, large headline setting hides stanfirst already and it's done with css. My assumption that scala would be a better way of hiding these elements is wrong. So this is consistent.